languages/javascript/scripts/scaffold/template/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@exercism/---",
+  "description": "---",
+  "author": "---",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/exercism/v3.git",
+    "directory": "languages/javascript/exercises/concept/---"
+  },
+  "devDependencies": {
```

Comment:
Is this the exact same we already use? If yes, you can just thumbs up this and resolve.

Reply:
Yep!